arm-software/linux/docs/StandardsSupport.md

```diff
@@ -0,0 +1,123 @@
+<!--===- docs/StandardsSupport.md
+
+   Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
Is this actually Part of the LLVM Project?

Reply:
This directory is outside llvm-project so will remove the banner.